SampleCode-V5/android-sdk-v5-sample/app-aircraft/src/main/java/dji/sampleV5/aircraft/control/PachKeyManager.kt

```diff
@@ -836,6 +905,7 @@ class PachKeyManager() {
                 Log.v("PachKeyManagerHIPPO", "Unknown Decision Check Failed")
                 break
             }
+            this@PachKeyManager.action = ""
```

Comment:
Does this simply reset the action string?

Reply:
Yes, when no specific action is being taken, the string is reset to empty.